Controller/TranslateController.php

```diff
@@ -26,7 +26,9 @@
 use JMS\TranslationBundle\Util\FileUtils;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Twig\Environment;
 
 /**
  * Translate Controller.
```

Comment:
Are you actually using the translations admin? I find completely unusable, if no one is using it, this controller with all it's routes should go.